ee/clickhouse/models/event.py

```diff
@@ -228,7 +228,7 @@ def get_agg_events_with_groups_count_for_teams_and_period(
         FROM events
         WHERE team_id IN (%(team_id_clause)s)
         AND timestamp between %(begin)s AND %(end)s
-        AND ($group_0 != '' OR $group_1 != '' OR $group_2 != '' OR $group_3 != '' OR $group_4 != '')
```

Comment:
Note: This was a spot missed in the previous PR

Reply:
Yes I did just enough to fix the tests an no more, leaving that to a separate followup.